src/spec/items.spec.js

```diff
@@ -115,6 +115,49 @@ describe('items', () => {
     res.body.should.be.an('array');
     res.body.length.should.eq(0);
   });
+  it('should return an item by a non-default key', async () => {
+    let res = await chai.request(server).get('/items/bronco.png?by=imageName');
+    res.should.have.status(200);
+    res.body.should.be.an('object');
+    Object.keys(res.body).length.should.be.greaterThan(0);
+  });
+  it('should return multiple items by a non-default key', async () => {
+    let res = await chai.request(server).get('/items/search/bronco.png?by=imageName');
+    res.should.have.status(200);
+    res.body.should.be.an('array');
+    res.body.length.should.be.greaterThan(0);
+  });
+  it('should return an item by nested keys', async () => {
+    let res = await chai.request(server).get('/items/rare?by=components.drops.rarity');
+    res.should.have.status(200);
+    res.body.should.be.an('object');
+    Object.keys(res.body).length.should.be.greaterThan(0);
```

Comment:
this doesn't really enforce much. i know this one makes it a bit weird, but maybe use a different key that's nested and can be verified for a particular value and verify other parts of the item that give confidence that the object is exactly what you expect. could just check the exact uniqueName

Reply:
Changed to new nested key that is only matched by two results, Akbronco and Bronco. Current code returns last found matching item so we can check for Bronco specifically. Multiple item return test changed to match this as well, testing for a specific sized array and checking the unique names of both items. See Commit de72f1c